lib/widgets/terminal.dart

```diff
@@ -0,0 +1,98 @@
+import 'package:flutter/material.dart';
+import 'package:zenith_monitor/constants/colors_constants.dart';
+import 'package:zenith_monitor/widgets/standard_app_bar.dart';
+
+/* 
+  todo: 
+  .Get value from textfield 
+  .Syncronize text with firebase storage
+*/
+
+class Terminal extends StatefulWidget {
+  const Terminal({Key? key});
+
+  @override
+  _TerminalState createState() => _TerminalState();
+}
+
+class _TerminalState extends State<Terminal> {
+  final textController = TextEditingController();
+
+  @override
+  void dispose() {
+    textController.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: const StandardAppBar(title: "Terminal"),
+      body: buildMainBody(),
+      backgroundColor: raisingBlack,
+    );
+  }
+
+  Center buildMainBody() {
+    return Center(
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.end,
+        children: [
+          terminalListView(),
+          terminalTextField(),
+        ],
+      ),
+    );
+  }
+
+  Expanded terminalListView() {
+    return Expanded(
+      child: ListView.builder(
+        itemCount: 30, //Ammount of data to be shown at the screen.
+        itemBuilder: (context, index) => Table(
+          columnWidths: const {
+            0: FractionColumnWidth(1/10),
+            1: FractionColumnWidth(9/10),
+          },children:[
+            TableRow(children:[
+              Padding(
+                padding: EdgeInsets.fromLTRB(20, 15, 0, 15),
+                child: Text('$index', style:TextStyle(color:gray)),
+              ),
+              Padding(
+                padding: EdgeInsets.fromLTRB(0, 15, 5, 15),
+                child: Text(" ", style: TextStyle(color: white),), // Here will be showed the text appering in the terminal.
+              )
```

Comment:
Existem algumas melhorias que o comando ``` flutter analyze``` sugere. Basicamente é só colocar a palavra *const* antes de determinados construtores. Vou colocar onde foi sugerido.
```suggestion
              Padding(
                padding: const EdgeInsets.fromLTRB(20, 15, 0, 15),
                child: Text('$index', style: const TextStyle(color:gray)),
              ),
              const Padding(
                padding: EdgeInsets.fromLTRB(0, 15, 5, 15),
                child: Text(" ", style: TextStyle(color: white),), // Here will be showed the text appering in the terminal.
              )
```

Para verificar se você fez corretamente, basta executar  o comando ```flutter analyze```. Outros erros vão aparecer por conta das alterações na **main**, se você quiser, você pode voltar a **main** para o jeito que estava e deixamos para fazer essas alterações durante o **merge** com a develop.